src/js/initializers/initStore.ts

```diff
@@ -19,16 +16,14 @@ function getInitialState(windowId) {
 export default async () => {
   const windowId = getUrlSearchParams().id
   const initialState = await getInitialState(windowId)
-  return createStore(
-    rootReducer,
-    initialState,
-    composeWithDevTools(
-      applyMiddleware(
-        reduxThunk.withExtraArgument({
-          globalDispatch,
-          createZealot
-        })
-      )
-    )
-  )
+  return configureStore({
```

Comment:
Docs for configure store: https://redux-toolkit.js.org/usage/usage-guide#simplifying-store-setup-with-configurestore